terraform/services/api-rds/main.tf

```diff
@@ -158,7 +103,7 @@ resource "aws_db_parameter_group" "v16_parameter_group" {
 resource "aws_db_instance" "api" {
   allocated_storage   = 500
   engine              = "postgres"
-  engine_version      = local.postgres_ver
+  engine_version      = 16
```

Comment:
Should we set this to 16.4? I see that's a change in the plan, and I think pinning to minor version is a good idea anyway.

Reply:
It actually doesn't pin to the minor version - we have the default auto_minor_version_upgrade setting (enabled) which is why the dbprotect scan findings of us being 1 revision back resolved themselves automatically on 11/5 (our first scheduled AWS maintenance window after 15.7/14.13 were released for auto-upgrade)

Setting 16 vs 16.4 isn't a meaningful gesture - the actual current version is stored in engine_version_current, which will remain on 16.4

Reply:
So this line in the plan will not result in a database reboot in the next maintenance window?

```
  # aws_db_instance.api will be updated in-place
  ~ resource "aws_db_instance" "api" {
      ~ engine_version                        = "16.4" -> "16"
```

Reply:
Correct